sw/device/silicon_creator/rom/e2e/immutable_rom_ext_section/BUILD

```diff
@@ -203,14 +222,14 @@ manifest(d = dicts.add(
 
 [
     opentitan_binary(
-        name = "immutable_rom_ext_section_test_{}".format(ld_script),
+        name = "immutable_rom_ext_section_test_{}".format(s["name"]),
         testonly = True,
         srcs = ["immutable_rom_ext_section_test.c"],
         exec_env = [
             "//hw/top_earlgrey:fpga_cw310_sival",
         ],
-        linker_script = "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_{}".format(ld_script),
-        manifest = ":manifest_addr_trans_on" if ld_script == "virtual" else ":manifest_addr_trans_off",
+        linker_script = "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_{}".format(s["slot"]),
+        manifest = ":manifest_addr_trans_on" if s["slot"] == "virtual" else ":manifest_addr_trans_off",
```

Comment:
This may not be required: the linker scripts in the code base (e.g. [ottf_common.ld](https://github.com/lowRISC/opentitan/blob/master/sw/device/lib/testing/test_framework/ottf_common.ld#L56-L58) now automatically detect whether address translation is enabled and create a linker-defined constant that is consumed by the [manifest definition](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/lib/manifest_def.c#L37).

You can probably eliminate the two explicit manifest rules in this BUILD file as well.

This address-translation auto-detect machinery was added to avoid having extraneous manifest definitions, particularly for ROM_EXT releases, but it should also help reduce complexity here.